setup.cfg

```
@@ -34,40 +34,35 @@ classifiers =
 [options]
 include_package_data = True
 packages = find:
-python_requires = >=3.9
+python_requires = >=3.12
 zip_safe = False
 install_requires =
-    invenio-assets>=2.0.0
-    invenio-i18n>=1.3.1
-    invenio-search-ui~=2.8.7
     opensearch-dsl>=2.0.0,<3.0.0
     opensearch-py>=2.0.0,<3.0.0
-    invenio-search-ui>=2.8.7
-    invenio-records-global-search>=0.2.0,<0.3.0
-    invenio-global-search>=0.2.0,<0.3.0
-    invenio-rdm-records ~=10.9.1
-    invenio-search-ui ~=2.8.7
-    invenio-records-lom>=0.12.0
-    invenio-records-marc21>=0.17.0
-    invenio-search[opensearch2]>=2.1.0,<3.0.0
-    invenio-app-rdm[opensearch2]~=12.0.0
+    invenio-global-search>=0.2.5
+    invenio-rdm-records >=18.0.0
+    invenio-app-rdm[opensearch2]>=13.0.0b3.dev2,<13.1.dev0
 
 [options.extras_require]
+lom =
+    invenio-global-search[lom]>=0.2.5
```

```suggestion
    invenio-global-search[lom]>=0.2.5,<0.3.0
 ```
 try to do here a version pinning, should ask Christoph if he has any dependency-driven versioning.
same for the **marc21**